examples/application/main.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"../../iis"
+)
+
+func main() {
+	log.Printf("Starting test...")
+
+	testAppPath := "C:\\inetpub\\testing"
+	testAppContent := "<html><head><title>IIS Testing</title></head><body><h1>Hello World!</h1></body></html>"
+	appPoolName := "testing"
+	applicationName := "testing"
+	websiteName := "Default Web Site"
+	testUrl := "http://localhost/testing"
+
+	err := createTestDirectory(testAppPath, testAppContent)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
if this is using the `run` method above - we should be able to make this:

```suggestion
		return fmt.Errorf("Error creating Test Directory: %s", err)
```